mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -605,11 +587,6 @@ struct ConvertVectorMaskedLoad final
   matchAndRewrite(vector::MaskedLoadOp op, OpAdaptor adaptor,
                   ConversionPatternRewriter &rewriter) const override {
 
-    // See #115653
-    if (op.getVectorType().getRank() != 1)
-      return rewriter.notifyMatchFailure(op,
-                                         "only 1-D vectors are supported ATM");
-
```

Comment:
Why do we remove the check? I thought that only load/store 1-D vectors is supported at this moment, and the patch does not extend that? (I don't see new n-D vector load/store tests being added in the PR.)

Reply:
I was wrong about this one! I confused the `ConstantMask`'s rank and the extract vector's rank. I reverted it back now.